### PR TITLE
Add machine branch for UI of makerManager

### PR DESCRIPTION
**Pull request by teojunjie**
Currently able to addMachine but unable to save data to xml file

### CODE REVIEW

File: src/main/java/seedu/address/commons/events/model/AddressBookChangedEvent.java [hunk 1]
```diff
@@ -14,6 +14,7 @@ public AddressBookChangedEvent(ReadOnlyAddressBook data) {
 
     @Override
     public String toString() {
-        return "number of persons " + data.getPersonList().size();
+        return "number of persons " + data.getPersonList().size()
+                + "\nnumber of machines " + data.getMachineList().size();
```
Comment (Psyf) [inline src/main/java/seedu/address/commons/events/model/AddressBookChangedEvent.java]: @teojunjie please revert, in light of upcoming changes discussed earlier today.

File: src/main/java/seedu/address/model/Model.java [hunk 2]
```diff
@@ -15,6 +15,9 @@
     /** {@code Predicate} that always evaluate to true */
     Predicate<Person> PREDICATE_SHOW_ALL_PERSONS = unused -> true;
 
+    /** {@code Predicate} that always eveluate to true for machines */
+    Predicate<Machine> PREDICATE_SHOW_ALL_MACHINES = unused -> true;
```
Comment (Psyf) [inline src/main/java/seedu/address/model/Model.java]: @teojunjie Can you please explain what this does?

Comment (teojunjie) [reply src/main/java/seedu/address/model/Model.java]: Actually i also not sure what this does

Comment (Psyf) [reply src/main/java/seedu/address/model/Model.java]: Can you put a //TODO near it so we later explore then?

File: src/main/java/seedu/address/commons/events/model/AddressBookChangedEvent.java (hunk 1 above)
Comment (teojunjie) [reply src/main/java/seedu/address/commons/events/model/AddressBookChangedEvent.java]: done

File: src/main/java/seedu/address/model/Model.java (hunk 2 above)
Comment (teojunjie) [reply src/main/java/seedu/address/model/Model.java]: Ok but you can merge now i suppose. I will add it to subsequent commits in the future

**Pull request merged**